.github/workflows/commit-validation.yml

```diff
@@ -6,15 +6,49 @@
 name: "Commit message validation"
 on:
   pull_request:
-    branches: [main, commitlint]
-    paths: [
-      # Commit validation in PRs applies to build-tools only as of now.
-      'build-tools/**', '.github/workflows/commit-validation.yml']
+    branches: [main, next]
     types: [opened, synchronize, edited, ready_for_review]
```

Comment:
Nit: maybe a brief comment explaining these type conditions for folks who aren't familiar.